cms/static/js/factories/import.js

```diff
@@ -87,30 +118,42 @@ define([
                 }
                 if (percentInt >= doneAt) {
                     bar.hide();
-                    // Start feedback with delay so that current stage of import properly updates in session
-                    setTimeout(
-                        function () { Import.startServerFeedback(feedbackUrl.replace('fillerName', file.name));},
-                        3000
-                    );
+
+                    // Start feedback with delay so that current stage of
+                    // import properly updates in session
+                    setTimeout(function () { Import.pollStatus(); }, 3000);
                 } else {
                     bar.show();
                     fill.width(percentVal).html(percentVal);
                 }
             },
-            done: function(event, data){
-                bar.hide();
-                window.onbeforeunload = null;
-                Import.displayFinishedImport();
-            },
-            start: function(event) {
-                window.onbeforeunload = function() {
-                    if (!Import.okayToNavigateAway) {
-                        return "${_('Your import is in progress; navigating away will abort it.')}";
-                    }
-                };
-            },
             sequentialUploads: true,
             notifyOnError: false
         });
+
+
+        var showImportSubmit = function (e) {
+            var filepath = $(this).val();
+
+            if (filepath.substr(filepath.length - 6, 6) === 'tar.gz') {
+                $('.error-block').hide();
+                $('.file-name').html($(this).val().replace('C:\\fakepath\\', ''));
+                $('.file-name-block').show();
+                chooseBtn.hide();
+                submitBtn.show();
+                $('.progress').show();
+            } else {
+                $('.error-block').html(gettext('File format not supported. Please upload a file with a <code>tar.gz</code> extension.')).show();
```

Comment:
Changing the string is sufficient within a pull request. The translation files are updated daily with a process outside of pull requests, to prevent translation file conflicts.